docs/setup.md

```diff
@@ -51,16 +51,27 @@ where `${CUDA_VERSION}` can be `80` (8.0), `90` (9.0), `92` (9.2), or `100` (10.
 
 Developers will be better served by pointing `$PYTHONPATH` to the root of the git-cloned source.
 
-### → on AWS...
+### → on AWS and Azure ...
 
-[AWS DeepLearning AMI](https://aws.amazon.com/amazon-ai/amis/) users only need to run the following line to install sockeye:
+[AWS DeepLearning AMI](https://aws.amazon.com/amazon-ai/amis/) users only need to run the following line to install sockeye (on an instance without a GPU):
 
 ```bash
-> sudo pip3 install sockeye --no-deps
+> conda create -n sockeye python=3.6
+> source activate sockeye
+> pip install sockeye --no-deps
 ```
 
-For other environments, you can choose between installing via pip or directly from source. Note that for the
-remaining instructions to work you will need to use `python3` instead of `python` and `pip3` instead of `pip`.
+On an instance with a GPU, the following commands will work
+
+```bash
+> conda create -n sockeye python=3.6
+> source activate sockeye
+> wget https://raw.githubusercontent.com/awslabs/sockeye/master/requirements/requirements.gpu-cu100.txt
+> pip install sockeye --no-deps -r requirements.gpu-cu100.txt
+rm requirements.gpu-cu100.txt
```

Comment:
Could you replace this with `${CUDA_VERSION}`?

Reply:
I did not check with all four versions. Shall I still specify all four or just the one that worked?

Reply:
I'd use the `CUDA_VERSION` environment variable and add a sentence as done in the pip section:
```
where ${CUDA_VERSION} can be 80 (8.0), 90 (9.0), 92 (9.2), or 100 (10.0).
```

Or maybe just once outside the pip and anaconda sections mention that `CUDA_VERSION` in the following refers to the cuda version present on the system.